library/dnet_common.c

```diff
@@ -396,17 +397,18 @@ void dnet_io_trans_alloc_send(struct dnet_session *s, struct dnet_io_control *ct
 	}
 
 	cmd->trans = t->rcv_trans = t->trans = atomic_inc(&n->trans);
+	dnet_get_backend_weight(t->st, cmd->backend_id, &backend_weight);
 	request_addr = dnet_state_addr(t->st);
 
 	dnet_log(n, DNET_LOG_INFO, "%s: created trans: %llu, cmd: %s, cflags: %s, size: %llu, offset: %llu, "
-			"fd: %d, local_offset: %llu -> %s weight: %f, wait-ts: %ld, ioflags: %s",
+			"fd: %d, local_offset: %llu -> %s weight: %f, backend_id: %d, wait-ts: %ld, ioflags: %s",
 			dnet_dump_id(&ctl->id),
 			(unsigned long long)t->trans,
 			dnet_cmd_string(ctl->cmd), dnet_flags_dump_cflags(cmd->flags),
 			(unsigned long long)ctl->io.size, (unsigned long long)ctl->io.offset,
 			ctl->fd,
 			(unsigned long long)ctl->local_offset,
-			dnet_addr_string(&t->st->addr), t->st->weight,
+		        dnet_addr_string(&t->st->addr), backend_weight, cmd->backend_id,
```

Comment:
We have undescribed rule for logs: if log is connected with certain node and its backend, this log should print node's address and backend_id in follow format: `{address}:{port}/{backend_id}`. Please, use this format here and wherever a log is connected with backend of certain node.